pihole-google.txt

```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
+m.doubleclick.net
+pagead46.l.doubleclick.net
+pubads.g.doubleclick.net
+s0.doubleclick.net
+secureframe.doubleclick.net
+securepubads.g.doubleclick.net
+static.doubleclick.netnod
+stats.g.doubleclick.net
+survey.g.doubleclick.net
+## Google-analytics.com
+google-analytics.com
+ssl.google-analytics.com
+www.google-analytics.com
+## Google.com
+admob.com
+ads.google.com
+adsense.com
+adservice.google.com
+ampcid.google.com
+assistant.google.com
+crashlytics2.l.google.com
+fitbit.com
+fundingchoices.google.com
+fundingchoicesmessages.google.com
+google.com
+googleadservices.com
+id.google.com
+pagead-googlehosted.l.google.com
+s0-2mdn-net.l.google.com
+video-stats.video.google.com
+www.assistant.google.com
+www.google.com
+www.googleadservices.com
+#Googleapis.com
+androidauto-pa.googleapis.com
+clientmetrics-pa.googleapis.com
+cloudconfig.googleapis.com
+dataprep.googleapis.com
+digitalassetlinks.googleapis.com
+discover-pa.googleapis.com
+mobilemaps-pa.googleapis.com
+people-pa.googleapis.com
+#Googletagmanager.com
+googletagmanager.com
+#Googletagservices.com
+googletagservices.com
+www.googletagservices.com
+#Googlesyndication
+googlesyndication.com
+pagead.googlesyndication.com
+pagead1.googlesyndication.com
+pagead2.googlesyndication.com
+pagead3.googlesyndication.com 
+#Googleusercontent.com
+displayads-formats.googleusercontent.com
+googlehosted.l.googleusercontent.com
+googleusercontent.com
+#More
+settings.crashlytics.com
+pagead2.googleadservices.com
+googleads.g.doubleclick.net
+datamixer-pa.googleapis.com
+instantmessaging-pa.googleapis.com
+mobilemaps-pa.googleapis.com
+cloudconfig.googleapis.com
+phonedeviceverification-pa.googleapis.com
+digitalassetlinks.googleapis.com
+mtalk.google.com
+alt1-mtalk.google.com
+alt2-mtalk.google.com
+alt3-mtalk.google.com
+alt4-mtalk.google.com
+alt5-mtalk.google.com
+alt6-mtalk.google.com
+alt7-mtalk.google.com
+alt8-mtalk.google.com
+app-measurement.com
+e.crashlytics.com
+firebaseinstallations.googleapis.com
+firebase-settings.crashlytics.com 
+tpc.googlesyndicytion.com
+androidauto-pa.googleapis.com
+clientmetrics-pa.googleapis.com 
+cloudconfig.googleapis.com 
+dataprep.googleapis.com 
+digitalassetlinks.googleapis.com 
+discover-pa.googleapis.com 
+mobilemaps-pa.googleapis.com
+people-pa.googleapis.com 
+e.crashlytics.com
+settings.crashlytics.com
+cryptauthdevicesync.googleapis.com
+deviceintegritytokens-pa.googleapis.com
+lamssettings-pa.googleapis.com
+mobilemaps-pa-gz.googleapis.com
+ulp1p-pa.googleapis.com
+geller-pa.googleapis.com
+play-lh.googleusercontent.com
+content-autofill.googleapis.com
+android-safebrowsing.google.com
+voledevice-pa.googleapis.com
+gmscompliance-pa.googleapis.com
+reports.crashlytics.com
+android.clients.google.com
+android.googleapis.com
+playatoms-pa.googleapis.com
+play.googleapis.com
+android.clients.google.com
+cloudconfig.googleapis.com
+app-measurement.com
+ssl.google-analytics.com
+connectivitycheck.android.com
+example.org
+ipv4only.arpa
```

Comment:
ipv4only.arpa domains are not linked to Google. These should be removed

Reply:
All changes made